criteria/pom.xml

```diff
@@ -52,34 +52,48 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <compilerId>javac-with-errorprone</compilerId>
-                    <forceJavacCompilerUse>true</forceJavacCompilerUse>
                     <compilerArgs>
-                        <arg>-Xep:CheckReturnValue:ERROR</arg>
-                        <arg>-Xep:MethodCanBeStatic:ERROR</arg>
-                        <arg>-Xep:BadImport:ERROR</arg>
-                        <arg>-Xep:MissingOverride:ERROR</arg>
-                        <arg>-Xep:OrphanedFormatString:ERROR</arg>
-                        <arg>-Xep:RedundantOverride:ERROR</arg>
-                        <arg>-Xep:RedundantThrows:ERROR</arg>
-                        <arg>-Xep:RemoveUnusedImports:ERROR</arg>
-                        <arg>-Xep:UnusedMethod:ERROR</arg>
+                        <arg>-XDcompilePolicy=simple</arg>
+                        <arg>-Xplugin:ErrorProne -Xep:CheckReturnValue:ERROR -Xep:MethodCanBeStatic:ERROR -Xep:BadImport:ERROR -Xep:MissingOverride:ERROR -Xep:OrphanedFormatString:ERROR -Xep:RedundantOverride:ERROR -Xep:RedundantThrows:ERROR -Xep:RemoveUnusedImports:ERROR -Xep:UnusedMethod:ERROR</arg>
```

Comment:
Is there any reason for merging these onto a single line? 

This documentation indicates that newlines inside a single `<arg></arg>` are a problem, but _presumably_ multiple are fine?

> Be aware that when running on JDK 8 the flags cannot be wrapped across multiple lines. JDK 9 and above do allow the flags to be separated by newlines. That is, the second <arg> element above can also be formatted as follows on JDK 9+, but not on JDK 8:
> ```xml
> <arg>
>   -Xplugin:ErrorProne
>   -Xep:DeadException:WARN
>   -Xep:GuardedBy:OFF
> </arg>
> 
> ```

https://errorprone.info/docs/flags

Reply:
(Author of that documentation here. ;) )

All `-Xplugin:ErrorProne` arguments must be specified inside the same `<arg>`. So on JDK 8 one either has to put them all on one line (like here) or hide the newlines inside XML comments, [like so](https://github.com/PicnicSupermarket/oss-parent/blob/master/pom.xml#L1039-L1079) (you can ignore the `<?SORTPOM` stuff).

More context in the original issue: google/error-prone#1115.

Reply:
On the same page, there's a note that JDK8 doesn't support multiline arguments. Unfortunately in order for this to work on JDK 8 they have to be inlined.